src/pip/_internal/req/req_uninstall.py

```diff
@@ -312,21 +313,25 @@ def __init__(self, dist: BaseDistribution) -> None:
         self._pth: Dict[str, UninstallPthEntries] = {}
         self._dist = dist
         self._moved_paths = StashedUninstallPathSet()
+        self._normalize_path_cached = functools.lru_cache(maxsize=256)(normalize_path)
```

Comment:
The maxsize definition is unnecessary (impossible to hit).

Reply:
p.s. worth adding a comment here to explain why the wrapping is implemented.

Reply:
> The maxsize definition is unnecessary (impossible to hit).

Without the `maxsize` definition, it caused errors in the Python 3.7 tests. I'm not sure, but I'm guessing that the simpler syntax only became possible in 3.8 due to this note? https://docs.python.org/3/whatsnew/3.8.html#functools

Reply:
Arguments are not required inside parentheses, you know.